Samples/WMCConsole_winrtcpp/WMCConsole_winrtcpp.cpp

```diff
@@ -0,0 +1,255 @@
+// 
+// WMCConsole_winrtcpp.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+#include <Windows.h>
+#include <iostream>
+#include <conio.h>
+#include <Mferror.h>
+#include <winrt\base.h>
+#include <winrt\Windows.Foundation.h>
+#include <winrt\Windows.Foundation.Collections.h>
+#include <winrt\Windows.Media.Capture.h>
+#include <winrt\Windows.Media.Core.h>
+#include <winrt\Windows.Media.Devices.h>
+#include <winrt\Windows.Media.MediaProperties.h>
+#include <winrt\Windows.Media.Capture.Frames.h>
+
+#include <winrt\Windows.Devices.Enumeration.h>
+#include <winrt\Windows.Graphics.Imaging.h>
+#include <winrt\Windows.Storage.h>
+#include <winrt\Windows.Storage.Streams.h>
+
+using namespace winrt;
+using namespace Windows::Media::Capture;
+using namespace Windows::Media::MediaProperties;
+using namespace Windows::Media::Capture::Frames;
+using namespace Windows::Foundation;
+using namespace Windows::Foundation::Collections;
+using namespace Windows::Devices::Enumeration;
+using namespace Windows::Storage;
+using namespace Windows::Graphics::Imaging;
+using namespace Windows::Media::Core;
+IVectorView<MediaFrameSourceGroup> sourceGroups;
+// Iterates through source groups and filter-out the frame sources which have IR,depth and other sources which we cannot consume in this app

```

Comment:
>//  [](start = 0, length = 3)

Add comment headers for all functions

Reply:
@Sudhaushu still an issue to resolve?

---
In reply to: [284523075](https://github.com/microsoft/Windows-Camera/pull/9#discussion_r284523075) [](ancestors = 284523075)

Reply:
working on it...

---
In reply to: [285753203](https://github.com/microsoft/Windows-Camera/pull/9#discussion_r285753203) [](ancestors = 285753203,284523075)